frontend/src/queries/schema/schema-surveys.ts

```diff
@@ -68,6 +68,7 @@ export interface SurveyDisplayConditionsSchema {
     selector?: string
     deviceTypes?: string[]
     deviceTypesMatchType?: SurveyMatchType
+    linkedFlagVariant?: string | null
```

Comment:
does it have to support `null`? all the other values are either the value or `undefined`

Reply:
removed